themes/Tokyonight-Dark/v1/theme.json

```diff
@@ -0,0 +1,329 @@
+      },
+      "success": {
+        "accent": "#4f8a10",
+        "paragraph": "#4f8a10",
+        "background": "#85ce68",
+        "border": "#e8e8e8",
+        "heading": "#4f8a10",
+        "icon": "#4f8a10",
+        "separator": "#e8e8e8",
+        "placeholder": "#a9a9a9",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      }
+    },
+    "statusBar": {
+      "primary": {
+        "accent": "#000000",
+        "paragraph": "#c0caf5",
+        "background": "#191a26",
+        "border": "#000000",
+        "heading": "#000000",
+        "icon": "#c0caf5",
+        "separator": "#000000",
+        "placeholder": "#000000",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      },
+      "secondary": {
+        "accent": "#000000",
+        "paragraph": "#c0caf5",
+        "background": "#191a26",
+        "border": "#000000",
+        "heading": "#000000",
+        "icon": "#c0caf5",
+        "separator": "#000000",
+        "placeholder": "#000000",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      },
+      "disabled": {
+        "accent": "#000000",
+        "paragraph": "#000000",
+        "background": "#000000",
+        "border": "#000000",
+        "heading": "#000000",
+        "icon": "#000000",
+        "separator": "#000000",
+        "placeholder": "#000000",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      },
+      "selected": {
+        "accent": "#000000",
+        "paragraph": "#000000",
+        "background": "#000000",
+        "border": "#000000",
+        "heading": "#000000",
+        "icon": "#000000",
+        "separator": "#000000",
+        "placeholder": "#000000",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      },
+      "error": {
+        "accent": "#000000",
+        "paragraph": "#000000",
+        "background": "#000000",
+        "border": "#000000",
+        "heading": "#000000",
+        "icon": "#ed333b",
+        "separator": "#c0caf5",
+        "placeholder": "#000000",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      },
+      "success": {
+        "accent": "#000000",
+        "paragraph": "#000000",
+        "background": "#000000",
+        "border": "#000000",
+        "heading": "#000000",
+        "icon": "#57e389",
+        "separator": "#000000",
+        "placeholder": "#000000",
+        "hover": "#000000",
+        "shade": "#000000",
+        "backdrop": "#000000",
+        "textSelection": "#000000"
+      }
+    },
+    "list": {
+      "primary": {
+        "accent": "#c0caf5",
```

Comment:
You will have to make this color a bit darker since the + icon is a little hard to see:

![image](https://github.com/streetwriters/notesnook-themes/assets/7473959/fb367403-3227-482d-a67d-cf3e127f5314)

I think we should automatically support darker icons in places like this.